packages/angular-material/src/other/master-detail/master.ts

```diff
@@ -140,9 +143,6 @@ export const removeSchemaKeywords = (path: string) => {
         top: 0;
         right: 0;
       }
-      .hide {
-        display: none;
```

Comment:
Why is this deleted?

Reply:
It might be an error when factoring out `@angular/flex-layout` with `[ngStyle]="{ display: hidden ? 'none' : '' }"` instead of `[fxHide]="hidden"`. Do you encounter any problem ?

Reply:
Yeah right, that's why I am writing. The delete button is always shown. We created a custom listWithDetail renderer to counter this problem. Is there a better solution? Or can you restore the deleted CSS class?

Reply:
I'm not a maintainer of this repo but I'd assume you can open an issue explaining the problem and them submit a PR to solve it. Then maintainers can maybe accept the PR and push it in the next release.

Reply:
Happy to review and merge a PR

Reply:
I've created the issue: https://github.com/eclipsesource/jsonforms/issues/2327
Should you give me permission in order to open a MR?

Reply:
@stanisev Thanks for creating the issue :) You can gladly open a MR anytime you want.

Reply:
Hi @stanisev , did you try to push the branch directly to this repository? This is not possible because you do not have write permissions. You need to fork the repository and push the branch to your fork. Then, you should be able to open a MR against this repository.